flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/CoGroupJoinITCase.java

```diff
@@ -234,6 +234,109 @@ public void invoke(String value) throws Exception {
 		Assert.assertEquals(expectedResult, testResults);
 	}
 
+
+	// TODO: design buffer join test
```

Comment:
@tillrohrmann Do you know why the checks have failed? There are 5 build jobs, only 3 of them passed. This is my first time to commit to an open source project. I have no idea how my code affects the failed tests.